src/Compatibility/ApiCompat/Microsoft.DotNet.PackageValidation/ApiCompatRunnerExtensions.cs

```diff
@@ -42,6 +42,12 @@ public static void QueueApiCompatFromContentItem(this IApiCompatRunner apiCompat
                 {
                     fallbackAssemblyReferences = rightPackage.FindBestAssemblyReferencesForFramework(leftTargetFramework);
                 }
+
+                // if we cannot find references for the left framework, then just use the same right references for the left assembly
+                if (fallbackAssemblyReferences is null && rightContentItems[0].Properties.TryGetValue("tfm", out tfmObj) && tfmObj is NuGetFramework rightTargetFramework)
+                {
+                    fallbackAssemblyReferences = rightPackage.FindBestAssemblyReferencesForFramework(rightTargetFramework);
```

Comment:
We might want to emit a diagnostic here.  So long as all the types that were in the more-specific framework are in the less specific framework and are forwarded from the assembly name (excluding version) roslyn will still load them.

Reply:
What do you mean by "less-specific" and "more-specific"?

Reply:
The direction of compatibility - newer versions being "more specific" also concrete frameworks being more specific than netstandard.  Maybe newer is close enough, just glossing over the netstandard difference.